### PR TITLE
[patch] Add support for GPU DRIVER VERSION environment var

### PR DESCRIPTION
**Pull request by caroazad**
Add gpu driver version environment variable so that users can set up whatever driver they want to use depending on os type and version.
Also added a variable for users to control the repository for the driver
Tested on roks and aws - passing so far

### CODE REVIEW

File: ibm/mas_devops/roles/nvidia_gpu/README.md
Comment (fyeh) [inline ibm/mas_devops/roles/nvidia_gpu/README.md:1]: @caroazad 
The deciding factor for which image version to use is the RHEL version. 
- RHEL 8.6
  - 470.103.01
- RHEL 7.9
  - 450.80.02

The version of RHEL that is used by OCP version is (per https://cloud.ibm.com/docs/openshift?topic=openshift-rhel_migrate)
- OCP 4.10+
  - RHEL 8
- OCP 4.9
  - Either RHEL 8 or RHEL 7 with 7 being the default
- OCP 4.6, 4.7, & 4.8
  - RHEL 7

Comment (caroazad) [reply ibm/mas_devops/roles/nvidia_gpu/README.md:1]: Thank you for this information. This is very helpful @fyeh

Comment (caroazad) [reply ibm/mas_devops/roles/nvidia_gpu/README.md:1]: @fyeh So this means that we can use 470.103.01 as default for 4.10 + and then the other ones can use 450.80.02?

File: ibm/mas_devops/roles/nvidia_gpu/README.md [hunk 1]
```diff
@@ -32,11 +32,27 @@ The channel to subscribe to for the gpu operator installation and updates. Avail
...
 
 ### gpu_driver_version
-The gpu driver version image that needs to be pulled from the gpu repository. It is recommended that the right version if GPU driver is used. The MVI installation documentation, the default version below should be used.
+The gpu driver version image that needs to be pulled from the gpu repository. It is recommended that the right version of GPU driver is used. The MVI installation documentation, the default version below should be used.
 
-- Environment Variable: none
-- Default Value: `470.103.01` if ocp version 4.11 and `450.80.02` otherwise
+- Environment Variable: `GPU_DRIVER_VERSION`
+- Default Value: `470.103.01` if ocp version 4.10+ and `450.80.02` otherwise
+The deciding factor for which image version to use is the RHEL version.
```
Comment (fyeh) [inline ibm/mas_devops/roles/nvidia_gpu/README.md:39]: change to RHEL or RHCOS version.

File: ibm/mas_devops/roles/nvidia_gpu/README.md [hunk 2]
```diff
@@ -32,11 +32,27 @@ The channel to subscribe to for the gpu operator installation and updates. Avail
...
+RHEL 8.6
+470.103.01
+RHEL 7.9
+450.80.02
+The version of RHEL that is used by OCP version is per https://cloud.ibm.com/docs/openshift?topic=openshift-rhel_migrate
+
+OCP 4.10+           --> RHEL 8
+OCP 4.9             --> Either RHEL 8 or RHEL 7 with 7 being the default
+OCP 4.6, 4.7, & 4.8 --> RHEL 7
+
```
Comment (fyeh) [inline ibm/mas_devops/roles/nvidia_gpu/README.md:50]: RHCOS 4.9 & 4.0
470.103.01
RHCOS 4.4 - 4.6
450.80.02

File: ibm/mas_devops/roles/nvidia_gpu/README.md [hunk 3]
```diff
@@ -32,9 +32,9 @@ The channel to subscribe to for the gpu operator installation and updates. Avail
 - Default Value: `v1.11`
 
 ### gpu_driver_version
-The gpu driver version image that needs to be pulled from the gpu repository. It is recommended that the right version if GPU driver is used. The MVI installation documentation, the default version below should be used.
+The gpu driver version image that needs to be pulled from the gpu repository. It is recommended that the right version of GPU driver is used. The MVI installation documentation, the default version below should be used.
```
Comment (fyeh) [inline ibm/mas_devops/roles/nvidia_gpu/README.md:35]: This does not read well:
```
The MVI installation documentation, the default version below should be used.

Comment (caroazad) [reply ibm/mas_devops/roles/nvidia_gpu/README.md]: I made some corrections. Please check for me. Thank you

**Review by fyeh — APPROVED** [review]
👍

**Review by durera — APPROVED** [review]

**Pull request merged**